install_add_pod/after/execution_output.txt

```diff
@@ -30,16 +30,22 @@ Downloading dependencies
  > Git download
      $ GIT_BIN clone https://github.com/tonymillion/Reachability.git PROJECT/Pods/Reachability --template= --single-branch --depth 1 --branch v3.1.0
      Cloning into 'PROJECT/Pods/Reachability'...
-     Note: checking out 'f7176f4798d068d233dca5223ae4bd9c8059e830'.
+     Note: switching to 'f7176f4798d068d233dca5223ae4bd9c8059e830'.
```

Comment:
cc @seanreinhardtapps any idea why these changed?

Reply:
This looks to be coincidental with Git 2.27.0:
https://github.com/git/git/commit/328c6cb853d7237098569de9f94bc3d259846a08

Reply:
Oh this might be caused by my local git version being different than what’s on CI

Thanks!